### PR TITLE
enable WebSocket pings

### PR DESCRIPTION
**Pull request by jtnord**
enable websocket pings to keep the connection open in case there is a firewall (which there is because well that's why you use the relay).

### CODE REVIEW

File: src/main/kotlin/Handler.kt [hunk 1]
```diff
@@ -79,6 +79,7 @@ class WebsocketHandler (val relayURI: String) {
             if (!receiver!!.connection.isOpen) {
                 return
             }
+            receiver!!.setConnectionLostTimeout(30) // enable 30s ping pongs
```
Comment (jtnord) [inline src/main/kotlin/Handler.kt:82]: this is the magic...  everything else was just to get to this point!

**Comment by jtnord** [comment]
this has been running on ops and astro for a while without any issues I think we can merge it.

**Comment by jtnord** [comment]
@mpailloncy @xino12

**Comment by mpailloncy** [comment]
Version 1.8 released

**Pull request merged**